src/App.tsx

```diff
@@ -1,25 +1,39 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
```

Comment:
The variable `items` is declared in the global scope of the module. It's better to move it inside the `App` component to avoid potential issues with state management and re-rendering. This is a code quality issue.